src/en/readallcomicscom/src/eu/kanade/tachiyomi/extension/en/readallcomicscom/ReadAllComics.kt

```diff
@@ -134,26 +145,20 @@ abstract class ReadAllComics(
     override fun searchMangaFromElement(element: Element) = SManga.create().apply {
         setUrlWithoutDomain(element.attr("href"))
         title = element.text().trim()
-        thumbnail_url = searchCover
+        thumbnail_url = "https://fakeimg.pl/200x300/?text=No%20Cover%0AOn%20Search&font_size=62"
     }
 
     override fun searchMangaSelector() = ".categories a"
     override fun searchMangaNextPageSelector() = null
 
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
-        title = document.select(mangaDetailsTitleSelector()).text().trim()
-        genre = document.select(mangaDetailsGenreSelector()).joinToString { it.text().trim() }
-        author = document.select(mangaDetailsAuthorSelector()).last()?.text()?.trim()
-        description = document.select(mangaDetailsDescriptionSelector()).text().trim()
-        thumbnail_url = document.select(mangaDetailsThumbnailSelector()).attr("abs:src")
+        title = document.select("h1").text().trim()
+        genre = document.select("p strong").joinToString { it.text().trim() }
```

Comment:
```suggestion
        genre = document.select("p strong").joinToString { it.text() }
```

`Element.text()` is trimmed already, [ref](https://jsoup.org/apidocs/org/jsoup/nodes/Element.html#text()).